jdaviz/configs/specviz/helper.py

```diff
@@ -229,6 +229,25 @@ def _redshift_listener(self, msg):
         if msg.param == "redshift":
             self._redshift = msg.value
 
+    def set_spectrum_tick_format(self, fmt, axis=None):
+        """
+        Manually set the tick format of one of the axes of the profile viewer.
+
+        Parameters
+        ----------
+        fmt : str
+            Format of tick marks in the spectrum viewer.
+        axis
+            The Specutils data axis.
```

Comment:
"specutils" does not quite make sense to me when I read it. I suggested something but if that is inaccurate, feel free to correct it.
```suggestion
            The spectrum viewer data axis.
            Axis 1 corresponds to the Y-axis and 0 to the X-axis.
```